src/js/game/LevelMVC/LevelView.js

```diff
@@ -608,14 +608,15 @@ module.exports = class LevelView {
 
     animation.onComplete.add(() => {
       this.activateUnpoweredRails(unpoweredRails);
-      this.playTrack(position, this.player.facing, isOnBlock, completionHandler);
+      this.playTrack(position, this.player.facing, isOnBlock, this.player, completionHandler);
     });
   }
 
-  playTrack(position, facing, isOnBlock, completionHandler) {
-    const track = this.controller.levelModel.actionPlane.getMinecartTrack(position, facing);
+  playTrack(position, facing, isOnBlock, entity = this.player, completionHandler) {
+    let  track = this.controller.levelModel.actionPlane.getMinecartTrack(position, facing);
```

Comment:
any reason for this change?

Reply:
Because when doing the redstone Rail work I change track to null so that the loop ends and the user stops when hitting uncharged rails. That's just a holdover from when I was ripping charged rail stuff out to get that darn integration test working